scenariolib/event_search.go

```diff
@@ -30,6 +31,18 @@ func newSearchEvent(e *JSONEvent, c *Config) (*SearchEvent, error) {
 	if se.query, validCast = e.Arguments["queryText"].(string); !validCast {
 		return nil, errors.New("Parameter query must be of type string in SearchEvent")
 	}
+	if e.Arguments["logEvent"] != nil {
```

Comment:
I like printing to Info, because someday I could log those to somewhere specific other than the console.

I have multiple logging level:
Trace.Println()
Info.Println()
Warning.Println()
Error.Printerr()

And I can easily decide where each of these loggers write to. For now they just all dump into console. But someday I could have different levels of logging go to different places.

I would say leave the Info logging.